editors/vscode/syntaxes/swarm.tmLanguage.json

```diff
@@ -148,7 +148,7 @@
       "patterns": [
         {
           "name": "variable.language.dir",
-          "match": "\\b(east|north|west|south|down|forward|left|back|right|)\\b"
+          "match": "\\b(east|north|west|south|down|forward|left|back|right)\\b"
```

Comment:
@kostmo, this typo you made in a30b6ed took me way too long to track down. 🤣

This empty pattern completely shuts down syntax highlighting after one token:
|Bugged|Fixed|
|---|---|
|<img width="624" alt="Screenshot 2024-06-09 at 13 16 49" src="https://github.com/swarm-game/swarm/assets/44544735/86480f09-2032-423e-aad0-4c1568230a6d">|<img width="624" alt="Screenshot 2024-06-09 at 13 17 36" src="https://github.com/swarm-game/swarm/assets/44544735/cd21a5c9-2ff3-4291-bb9b-dde0d226bf59">|

Reply:
Oof, great catch!